### PR TITLE
Fix minor typos and formatting

### PR DESCRIPTION
**Pull request by jrnold**
- fix misspellings
- format all R package names so they are bold
- format all R objects and functions to be typewriter font
- "dataframe" to "data frame"

### CODE REVIEW

**Comment by tomwright01** [comment]
This is great, thanks @jrnold for proof reading.
There does seem to be a merge conflict in 04-data-structures-part1.Rmd. Also with the number of outstanding PRs I'd like to hold this until the repository settles down a bit.

File: 01-rstudio-intro.Rmd [hunk 1]
```diff
@@ -122,7 +122,7 @@ complete it:
 
 Any time you hit return and the R session shows a "+" instead of a ">", it
 means it's waiting for you to complete the command. If you want to cancel
-a command you can simply hit "Esc" and RStudio will give you back the ">"
+a command you can simply hit <kbd>Esc</kbd> and RStudio will give you back the ">"
```
Comment (gvwilson) [inline 01-rstudio-intro.Rmd]: We prefer simple back quotes to HTML `<kbd>` tags.

File: 04-data-structures-part1.Rmd [hunk 2]
```diff
@@ -104,9 +105,13 @@ Oh no, our weights aren't the double type anymore! If we try to do the same math
 cats$weight[1] + cats$weight[2]
 ```
 
+<<<<<<< a9dc3d64bcc69800f7f6e56a703f9afb0062382b
```
Comment (gvwilson) [inline 04-data-structures-part1.Rmd]: This looks like you're trying to commit a merge conflict?

File: 04-data-structures-part1.Rmd [hunk 3]
```diff
@@ -104,9 +105,13 @@ Oh no, our weights aren't the double type anymore! If we try to do the same math
...
 What happened? When R reads a csv into one of these tables, it insists that everything in a column be the same basic type; if it can't understand *everything* in the column as a double, then *nobody* in the column gets to be a double. The table that R loaded our cats data into is something called a *data.frame*, and it is our first example of something called a *data structure* - things that R knows how to build out of the basic data types. In order to successfully use our data in R, we need to understand what these basic data structures are, and how they behave. For now, let's return to our original cats data and reload it, while we investigate this behavior further:
+=======
```
Comment (gvwilson) [inline 04-data-structures-part1.Rmd]: As above.

File: 06-data-subsetting.Rmd [hunk 4]
```diff
@@ -409,7 +409,7 @@ f[-3]
 
 ## Matrix subsetting
 
-Matrices are also subsetted using the `[` function. In this case
+Matrices are also subset using the `[` function. In this case
```
Comment (gvwilson) [inline 06-data-subsetting.Rmd]: Original is correct - using 'subset' as a verb, so 'subsetted'.

File: 08-plot-ggplot2.Rmd [hunk 5]
```diff
@@ -35,16 +35,16 @@ package, and the [ggplot2][ggplot2] package.
 [lattice]: http://www.statmethods.net/advgraphs/trellis.html
 [ggplot2]: http://www.statmethods.net/advgraphs/ggplot2.html
 
-Today we'll be learning about the ggplot2 package, because
+Today we'll be learning about the **ggplot2** package, because
```
Comment (gvwilson) [inline 08-plot-ggplot2.Rmd]: Back quotes (to put in typewriter case) rather than bold, since it's a library name.

File: 08-plot-ggplot2.Rmd [hunk 6]
```diff
@@ -35,16 +35,16 @@ package, and the [ggplot2][ggplot2] package.
...
 it is the most effective for creating publication quality
 graphics.
 
-ggplot2 is built on the grammar of graphics, the idea that any plot can be
-expressed from the same set of components: a **data** set, a
-**coordinate system**, and a set of **geoms**--the visual representation of data
+**ggplot2** is built on the grammar of graphics, the idea that any plot can be
```
Comment (gvwilson) [inline 08-plot-ggplot2.Rmd]: As above - we don't use bold for library names elsewhere.

File: 08-plot-ggplot2.Rmd [hunk 7]
```diff
@@ -292,7 +292,7 @@ code to modify!
 > In the previous examples and challenge we've used the `aes` function to tell
 > the scatterplot **geom** about the **x** and **y** locations of each point.
 > Another *aesthetic* property we can modify is the point *color*. Modify the
-> code from the previous challenge to **color** the points by the "continent"
```
Comment (gvwilson) [inline 08-plot-ggplot2.Rmd]: Why italics or bold? We don't use these stylings elsewhere.

Comment (jrnold) [inline 08-plot-ggplot2.Rmd]: I was following the standard convention in R documentation and the Journal of Statistical Softaware that functions are typeset in `monospace` and R packages are in **bold**.

**Comment by jrnold** [comment]
This now has too many conflicts, and my changes were not all in keeping with conventions elsewhere in the lesson, so I'm closing this.

**Pull request closed without merging**